sdk/servicebus/service-bus/test/browserified/testUtils.ts

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
+
+import {
+  SendableMessageInfo,
+  QueueClient,
+  TopicClient,
+  ServiceBusClient,
+  SubscriptionClient,
+  ReceiveMode,
+  ServiceBusMessage
+} from "../../src";
+
+import { delay } from "rhea-promise";
+
+export class TestMessage {
+  static sessionId: string = "my-session";
+
+  // @ts-ignore
+  static getSample(): SendableMessageInfo {
+    const randomNumber = Math.random();
+    return {
+      body: `message body ${randomNumber}`,
+      messageId: `message id ${randomNumber}`,
+      partitionKey: `dummy partition key`,
+      contentType: `content type ${randomNumber}`,
+      correlationId: `correlation id ${randomNumber}`,
+      timeToLive: 60 * 60 * 24,
+      label: `label ${randomNumber}`,
+      to: `to ${randomNumber}`,
+      replyTo: `reply to ${randomNumber}`,
+      scheduledEnqueueTimeUtc: new Date(),
+      userProperties: {
+        propOne: 1,
+        propTwo: "two",
+        propThree: true
+      }
+    };
+  }
+
+  // @ts-ignore
```

Comment:
Which error are we ignoring here? There are a lot of `ts-ignore`s in this file.
I understand the need for `window.__env__`, but I don't get the need for others

Reply:
Ah yes, I had them in for case when I was importing the browserified SDK directly. Removed these

Reply:
There are 2 `ts-ignore` still for `SendableMessageInfo` and `ServiceBusMessage` inside the `checkMessageContents()` method